app/views/users/edit.html.erb

```diff
@@ -1,13 +1,9 @@
-<h2>Please enter your SAM and/or D.U.N.S id</h2>
+<h2>Please enter your DUNS Number:</h2>
 
-<form action="/users/<%= @user.id %>" method='POST'>
-  <input type='hidden' name='_method' value='put'>
-  <label for="sam_id">SAM id:</label>
-  <input id="sam_id" name="sam_id" type="text" value="<%= @user.sam_id %>">
+<%= form_for @user, method: :put do |f| %>
 
-  <label for="duns_id">D.U.N.S id:</label>
-  <input id="duns_id" name="duns_id" type="text" value='<%= @user.duns_id %>'>
-
-  <input type='submit' value='Save'>
-</form>
+  <%= f.label :duns_id, 'DUNS Number' %>
```

Comment:
since this is called "DUNS Number" in the view I wonder if it should be called `duns_number` in the database. When I see `id`, I always assume it is a unique id for another model. (started looking for a `duns` model here). Sometimes using `xyz_id` can make sense anyway, but since it seems like users will be calling it "Duns number" anyway, why not call it that on the backend anyway!

naming is fun! :8ball: